### PR TITLE
fix timeout unit mismatch issue

### PR DESCRIPTION
**Pull request by junchaowu**

### CODE REVIEW

File: tchannel/tchannel.py [hunk 1]
```diff
@@ -268,7 +268,7 @@ def advertise(self, routers, name=None, timeout=None):
             other services will use to make contact with you.
 
         :param timeout:
-            The timeout (in seconds) for the initial advertise attempt.
+            The timeout (in sec) for the initial advertise attempt.
```
Comment (abhinav) [inline tchannel/tchannel.py]: so this one is unchanged?

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,13 @@
 Changes by Version
 ==================
 
+0.16.7 (unreleased)
+-------------------
+
+- Fixed a bug that the time unit for ttl is unclear. The unit for ttl is set
```
Comment (blampe) [inline CHANGES.rst]: Not just unclear, timeouts were broken.

File: tchannel/tchannel.py (hunk 1 above)
Comment (blampe) [inline tchannel/tchannel.py]: Keep as seconds.

Comment (junchaowu) [inline tchannel/tchannel.py]: this one is advertise. Advertise timeout by default is 30s.

Comment (blampe) [inline tchannel/tchannel.py]: Can you confirm it's passing it as 30000?

File: CHANGES.rst [hunk 3]
```diff
@@ -1,6 +1,13 @@
...
+  to millisecond.
```
Comment (abhinav) [inline CHANGES.rst]: > Fixed a time unit conversion bug that was treating timeouts incorrectly.

**Pull request closed without merging**